Automattic/wp-calypso
# Jetpack Plans Cancellation Survey: Replace text input with dropdown list of downgradeable plans.

### Changes proposed in this Pull Request

This PR adds a select box dropdown for the "I'd like to downgrade to another plan" cancellation reason in the Jetpack Plans Cancellation Survey.

Please see 5058-gh-Automattic/jpop-issues for more context.

Fixes 1164141197617539-as-1199561096729181

### Testing instructions

* Run this PR (Calypso Blue).
* Select a Jetpack site that has a paid plan.
* Go to the Billing section located at `/purchases/subscriptions/:site`.
* Select the site's plan.
* Click on the `[Remove :planName]` button at the bottom of the section.
* Select the radio option labeled as `I'd like to downgrade to another plan`.
* Verify you see  a select box dropdown containing downgradable plans and products.
* Verify the select box dropdown is working correctly.
* Repeat these steps but selecting different sites with different plans. (different plans with have different available downgradable plans to select, for example, a site with Jetpack Complete with display the option to downgrade to Jetpack Security Realtime, or Jetpack Security Daily, and individual products. However a site with Jetpack Security Daily won't show any "plans" to downgrade to, only individual products. (because Jetpack Security is the lowest tier "plan" already and there does not exist a 'lower' "plan".)

### Screenshots

![Markup 2020-12-11 at 13 37 33](https://user-images.githubusercontent.com/11078128/101958045-6c7b3580-3bb7-11eb-8753-587f5e8a6e3a.png)

![Markup 2020-12-11 at 13 40 12](https://user-images.githubusercontent.com/11078128/101958066-713fe980-3bb7-11eb-9526-85620f6b4e3c.png)

![Markup 2020-12-11 at 13 46 21](https://user-images.githubusercontent.com/11078128/101958075-74d37080-3bb7-11eb-80b3-19da73c89f4d.png)
